app/renderer/components/downloadItem.js

```diff
@@ -74,10 +82,17 @@ class DownloadItem extends ImmutableComponent {
     return <span
       onContextMenu={contextMenus.onDownloadsToolbarContextMenu.bind(null, this.props.downloadId, this.props.download)}
       onDoubleClick={this.onOpenDownload}
+      onMouseLeave={this.onHideDeleteConfirmation}
       className={cx({
         downloadItem: true,
+        deleteConfirmationVisible: this.props.downloadDeleteConfirmationVisible,
         [this.props.download.get('state')]: true
       })}>
+      {
+        this.props.downloadDeleteConfirmationVisible
```

Comment:
You could also do `this.props.downloadDeleteConfirmationVisible && <Component ... />`